src/main/flight/mixer.c

```diff
@@ -348,7 +348,7 @@ void mixerUseConfigs(
 }
 
 #ifdef USE_SERVOS
-int16_t determineServoMiddleOrForwardFromChannel(servoIndex_e servoIndex)
+int16_t determineServoMiddleOrForwardFromChannel(servoChannel_e servoIndex)
```

Comment:
it doesn't make sense, to me, to have one called channel and the other called index.

`servoChannel_e servoChannel` ?